libs/utils/decorate.js

```diff
@@ -308,3 +306,21 @@ export function decorateMultiViewport(el) {
   }
   return foreground;
 }
+
+export function decorateAnchorVideo({ src, anchorTag }) {
+  if (anchorTag.closest('.marquee, .aside, .hero-marquee') && !anchorTag.hash) anchorTag.hash = '#autoplay';
```

Comment:
Any need for some extra failsafe mechanisms?
```suggestion
export function decorateAnchorVideo({ src, anchorTag } = {}) {
  if (!src.length || !anchorTag instanceof HTMLElement) return; // or some other relevant action
  if (anchorTag.closest('.marquee, .aside, .hero-marquee') && !anchorTag.hash) anchorTag.hash = '#autoplay';
```